assets/app/view/game_card.rb

```diff
@@ -158,6 +169,20 @@ def render_optional_rules
       h(:div, [h(:strong, 'Optional Rules: '), rendered_rules])
     end
 
+    def render_invite_link
+      return unless new? && owner?
```

Comment:
change to if statement

Reply:
That would trigger Rubocop Style/GuardClause.